tests/chainer_tests/test_computational_graph.py

```diff
@@ -191,4 +191,26 @@ def test_tail_node(self):
         _check(self, (self.y, ), 9, 8)
 
 
+class TestGraphBuilder8(unittest.TestCase):
```

Comment:
The objective of this test fixture is different from previous ones. So could you change the name of this class?